platform/security/certificate/security-certificate-generator/src/test/java/org/codice/ddf/security/certificate/generator/CertificateCommandTest.java

```diff
@@ -0,0 +1,228 @@
+import org.bouncycastle.asn1.x509.GeneralName;
+import org.bouncycastle.asn1.x509.GeneralNamesBuilder;
+import org.bouncycastle.cert.X509CertificateHolder;
+import org.hamcrest.MatcherAssert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import ddf.security.SecurityConstants;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CertificateCommandTest {
+    private static final String[] SANs = new String[] {"IP:1.2.3.4", "DNS:A"};
+    private static final String SANs_ARG = "IP:1.2.3.4,DNS:A";
+
+    @Rule
+    public TemporaryFolder temporaryFolder = new TemporaryFolder();
+
+    File systemKeystoreFile = null;
+
+    private static void validateCertificateHasNoSAN(KeyStoreFile ksf, String alias)
+            throws Exception {
+        final KeyStore.Entry ke = ksf.getEntry(alias);
+
+        assertThat(ke, instanceOf(KeyStore.PrivateKeyEntry.class));
+        final KeyStore.PrivateKeyEntry pke = (KeyStore.PrivateKeyEntry) ke;
+        final Certificate c = pke.getCertificate();
+        final X509CertificateHolder holder = new X509CertificateHolder(c.getEncoded());
+
+        MatcherAssert.assertThat("There should be no subject alternative name extension",
+                holder.getExtension(org.bouncycastle.asn1.x509.Extension.subjectAlternativeName),
+                nullValue(org.bouncycastle.asn1.x509.Extension.class));
+    }
+
+    private static void validateCertificateHasSANs(KeyStoreFile ksf, String alias)
+            throws Exception {
+        final KeyStore.Entry ke = ksf.getEntry(alias);
+
+        assertThat(ke, instanceOf(KeyStore.PrivateKeyEntry.class));
+        final KeyStore.PrivateKeyEntry pke = (KeyStore.PrivateKeyEntry) ke;
+        final Certificate c = pke.getCertificate();
+        final X509CertificateHolder holder = new X509CertificateHolder(c.getEncoded());
+        final org.bouncycastle.asn1.x509.Extension csn =
+                holder.getExtension(org.bouncycastle.asn1.x509.Extension.subjectAlternativeName);
+
+        MatcherAssert.assertThat(csn.getParsedValue()
+                        .toASN1Primitive()
+                        .getEncoded(ASN1Encoding.DER),
+                equalTo(new GeneralNamesBuilder().addName(new GeneralName(GeneralName.iPAddress,
+                        "1.2.3.4"))
+                        .addName(new GeneralName(GeneralName.dNSName, "A"))
+                        .build()
+                        .getEncoded(ASN1Encoding.DER)));
+    }
+
+    private static void validateKeyStore(String alias, boolean hasSAN) throws Exception {
+        final KeyStoreFile ksf = CertificateCommand.getKeyStoreFile();
+
+        assertThat(ksf.aliases()
+                .size(), is(2));
+        if (hasSAN) {
+            validateCertificateHasSANs(ksf, alias);
+        } else {
+            validateCertificateHasNoSAN(ksf, alias);
+        }
+    }
```

Comment:
I am running the formatter and because they are static, it leaves them at the top